app/src/main/java/com/example/uyen/recyclerview/Hero.java

```diff
@@ -0,0 +1,29 @@
+package com.example.uyen.recyclerview;
+
+public class Hero {
+    private String mName;
+    private int mImage;
+
+    public Hero(String mName, int mImage) {
```

Comment:
em làm theo tip trick số 6
https://viblo.asia/p/mot-so-tips-tricks-khi-lam-viec-voi-android-studio-yMnKMJjgZ7P
sau đó xóa constructor, getter setter đi gen lại nhé
sau khi làm lại thì code sẽ ntn
```suggestion
    public Hero(String name, int image) {
```